lms/djangoapps/django_comment_client/management/commands/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+'''
+Tests for management command line tools
+'''
+__author__ = 'azimpradhan'
```

Comment:
Actually, I'm cool with the docstring; let's drop the `__author__` line, though.